README.md

```diff
@@ -52,7 +53,37 @@ Example command to spin up 2 Windows nodes and retain them after test run:
 ```
 hack/run-ci-e2e-test.sh -s -k "openshift-dev" -n 2      
 ```
+
+## Running locally with OLM 
+
+### Create Secrets
+Create the cloud provider and cloud private key secrets. The cloud-private-key should match the keypair used in the
+Windows Machine Config CR you will use.
+```shell script
+# Change paths as necessary
+oc create secret generic cloud-credentials --from-file=credentials=/home/$USER/.aws/credentials
+oc create secret generic cloud-private-key --from-file=private-key.pem=/home/$USER/.ssh/$keyname
+```
+
+Put the kubeconfig you wish to use in a secret. In order to use the permissions of the windows-machine-config-operator
+service account the kubeconfig should be generated from the service account.
+```shell script
+# Change paths as necessary
+oc create secret generic kubeconfig --from-file=kubeconfig=/path/to/kubeconfig
+```
+
+### Testing the operator locally with OLM
```

Comment:
This should be under `Deploying the operator on a local cluster` with creating secrets being a common section. In the `Bundling the Windows Machine Config Operator`, link to `Testing the operator locally with OLM` saying you can skip creating the bundle if you want to run locally without creating images.